src/test/regress/sql/grant_role_2pc.sql

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
Can we remove all unnecessary newlines in this test file?

Reply:
Fixed